datastream-kafka-connector/src/test/java/com/linkedin/datastream/connectors/kafka/mirrormaker/TestKafkaMirrorMakerConnector.java

```diff
@@ -697,6 +698,44 @@ public void testChangeDatatstreamAssignment() throws Exception {
 
   }
 
+  @Test
+  public void testGetStateForMultipleTasks() throws Exception {
+    String yummyTopic = "YummyPizza";
```

Comment:
nit: you can just store these strings in an array to reduce repetition:
```java
    String[] topics = {"YummyPizza", "SaltyPizza", "HealthySalad"};
    Arrays.stream(topics).forEach(t -> createTopic(_zkUtils, t));
```
or in a list
```java
    List<String> topics = Arrays.asList("YummyPizza", "SaltyPizza", "HealthySalad");
    topics.forEach(t -> createTopic(_zkUtils, t));
```